project/src/types/mocksTypes.ts

```diff
@@ -0,0 +1,51 @@
+export type UserType = {
+  avatarUrl: string;
+  id: string;
+  isPro: boolean;
+  name: string;
+}
+
+export type ReviewType = {
+    comment: string;
+    date: string;
+    id: string;
+    rating: number;
+    user: UserType;
+}
+
+export type LocationType = {
+  latitude: number;
+  longitude: number;
+  zoom: number;
+}
+
+export type CityType = {
+  location: LocationType;
+  name: string;
+}
+
+export enum HoustinTypeEnum {
+  APARTAMENT = 'apartment',
+  ROOM = 'room',
+  HOUSE = 'house',
+  HOTEL = 'hotel'
```

Comment:
```suggestion
export enum HoustinTypeEnum {
  APARTAMENT,
  ROOM,
  HOUSE,
  HOTEL
```

тебе генерируется автоматически объект где ключи и значение будут равны друг друга и будут капсом, раз тебе надо не капсом, просто в коде применишь функции toLowerCase()